include/avif/avif.h

```diff
@@ -210,22 +210,29 @@ AVIF_API const char * avifResultToString(avifResult result);
 // ---------------------------------------------------------------------------
 // avifHeaderFormat
 
+// Combinatorial flag for selecting container strategies when encoding an image.
```

Comment:
Does this mean the combination of `AVIF_HEADER_MINI | AVIF_HEADER_EXTENDED_PIXI` is legal?

If `AVIF_HEADER_MINI | AVIF_HEADER_EXTENDED_PIXI` is illegal, I don't see any code that checks for illegal combinations of `avifHeaderFormat` flags.

Reply:
> Does this mean the combination of `AVIF_HEADER_MINI | AVIF_HEADER_EXTENDED_PIXI` is legal?

Yes.